app/controllers/api_controller.rb

```diff
@@ -108,6 +109,16 @@ def render_unauthorized_error(exception)
     )
   end
 
+  def connection_error(exception)
+    render(
+      json: { errors: [{
+        title: 'Connection Error',
```

Comment:
Should this be a Nomis connection Error?

Reply:
good point, the reality is that we don't know, since we could use faraday to connect to different services ... 
to fix this I'm passing a message when I raise the exception, this is the response:
 
`{"errors"=>[{"title"=>"Connection Error", "detail"=>"Faraday::TimeoutError: Nomis Connection Error"}]}`

that specifies `Nomis`